src/app/user/register/index.njk

```diff
@@ -32,48 +31,48 @@
       <div class="govuk-form-group {{ m.form_group_class('userFname', errors) }}">
         <label class="govuk-label govuk-label--m" for="userFname">{{ __('field_given_name') }}</label>
         {{ m.error_message('userFname', errors) }}
-        <input autocomplete="off" class="govuk-input govuk-input--width-30"
-        data-val="true" data-val-required="The given name field is  required." id="userFname" name="userFname" style="width:100%"
```

Comment:
data-val stuff appears to be unobtrusive validation I think it's called, something jquery uses and stuff, or ASP.net.

I don't see jquery-validate imported into this app, and we know how the current validation works as it is, so this seems to be some hangover from another time.